clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -802,31 +784,31 @@ private void maybeRegisterTopicMetrics(String topic) {
                 Map<String, String> metricTags = Collections.singletonMap("topic", topic);
 
                 topicRecordCount = this.metrics.sensor(topicRecordsCountName);
-                MetricName rateMetricName = this.metrics.metricInstance(metricsRegistry.topicRecordSendRate, metricTags);
-                MetricName totalMetricName = this.metrics.metricInstance(metricsRegistry.topicRecordSendTotal, metricTags);
+                MetricName rateMetricName = this.metrics.topicRecordSendRate(metricTags);
```

Comment:
The ones above are accessing a variable, whereas this is calling a method. This is needed because I need to pass in the tags.